config/routes.rb

```diff
@@ -20,6 +22,10 @@
   get '/api/totals/total_entries' => 'entries#total', as: 'entries_total'
   get '/admin', to: 'main#admin'
   get '/admin/*path', to: 'main#admin'
+  get '/teams', to: 'teams#index', as: 'teams'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.